coalib/parsing/DefaultArgParser.py

```diff
@@ -119,6 +120,26 @@ def default_arg_parser(formatter_class=None):
                                 metavar='BOOL',
                                 help='Write the logs as json to a file '
                                 'where filename is specified as argument.')
+    if parser_type == 'coala-html':
```

Comment:
@tushar-rishav he's right, if we want to use the arg parser cleanly we have to move coala-html into coala or add the arguments outside...